library/Zend/XmlRpc/Server.php

```diff
@@ -492,6 +492,9 @@ public function getSystem()
      * how to handle arguments. If set to true, all methods including constructor
      * will receive the arguments. If set to false, only constructor will receive the
      * arguments
+     *
+     * @param  bool|null $flag
+     * @return Server
```

Comment:
You can use `@return self` here, just like you did on the other places where `return $this;` is used.